x/posts/internal/types/msgs_test.go

```diff
@@ -238,9 +238,9 @@ func TestMsgUnlikePost_ValidateBasic(t *testing.T) {
 		error sdk.Error
 	}{
 		{
-			name:  "Invalid post id returns error",
+			name:  "Invalid post newID returns error",
 			msg:   types.NewMsgUnlikePost(types.PostID(0), testOwner),
-			error: sdk.ErrUnknownRequest("Invalid post id"),
+			error: sdk.ErrUnknownRequest("Invalid post newID"),
```

Comment:
I think this is a refactoring error

```suggestion
			error: sdk.ErrUnknownRequest("Invalid post id"),
```